src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -169,69 +178,70 @@ private static void checkForApiType(Properties properties) {
 				} 
 			}
 		} catch (JsonParseException e) {
```

Comment:
Done. i get to learn about maven today because of this comment. maven default java type compile 1.5( doesn't matter what you machine shows `mvn -v`), so my build was failing with his suggestion. The only work around is to exclusively set the java compile level . I have set it to Java 1.8